row.go

```diff
@@ -407,24 +407,24 @@ type levels struct {
 // individually as the base case.
 type deconstructFunc func([][]Value, levels, reflect.Value)
 
-func deconstructFuncOf(columnIndex int16, node Node) (int16, deconstructFunc) {
+func deconstructFuncOf(columnIndex int16, node Node, source tagSource) (int16, deconstructFunc) {
```

Comment:
How about naming the new argument `tags` instead of `source`, this might better communicate what the value is.

Reply:
done 1f6c6bd